AcademyResidentInformationApi.Tests/AcademyResidentInformationApi.Tests.csproj

```diff
@@ -1,26 +1,25 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
-    <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
```

Comment:
I think this is the code analysis that @Renny4Real added, so don't think it should be deleted

Reply:
It doesn't compile with the settings